components/03-organisms/menu/utility-nav/utility-nav.twig

```diff
@@ -9,7 +9,14 @@
   } %}
   {# Search #}
   {% if utility_nav__search %}
-    <input type="text" placeholder="Search this site">
+    <div {{ bem('search', [], utility_nav__base_class) }}>
+      {% include "@atoms/images/icons/_icon.twig" with {
+        icon__name: 'magnifying-glass',
+        icon__blockname: utility_nav__base_class,
+        icon__modifiers: ['search'],
```

Comment:
Oooooo, it is here that we would set:
icon__decorative: true
or 
icon__role: img

I get it now!